docs/camino-messenger/introduction.md

```diff
@@ -0,0 +1,156 @@
+- **network_fee**: Contains specifications related to network transaction fees.
+- **product_list**: List of products or services available from a provider
+- **product_details**: Description of products or services available from a provider
+- **accommodation**: Defines the message type for Accommodations like hotels and holiday homes.
+- **transport**: Defines the message type for Flights, Rail and Transfer.
+- **activity**: for Tickets & Excursions
+- **cruise**: Cruise (not started)
+- **car_rental**: Rent a car, Rent a Camper
+- **insurance**: Insurances (not started)
+- **camping**: Camping (not started)
+- **package**: Packages, inherent format consisting of the above structures for the services/products included in the package
+- **flight_status**: Flight status information
+- **entry_destination**: Entry requirements & Destination information
+
+## Nested Messages - Data Types
+
+Delving deeper into the data structures and components, protobuf uses shared structures as nested messages, which we call **"types"**. For example the traveller details in traveller.proto. The transformation to your native logic only requires a one-time development for all main Message Types that will be implemented. Nested messages provide a clear, hierarchical structure, making it easier to understand, review and discuss improvements to the Message standard with the Camino Network Partners. Other benefits of the nested structure of Protobuf are the encapsulation leading to cleaner and easier maintainable code and type safety which helps catching errors early in the integration process.
+
+Some examples:
+
+- **coordinate.proto**: Captures geographical coordinates.
+- **currency.proto**: Handles different currency types and associated attributes.
+- **date.proto**: A flexible schema for managing date-related data.
+- **distance.proto**: Quantifies and categorizes distances, catering to various units and interpretations.
+- **filter.proto**: Offers dynamic and static filtering capabilities.
+- **geo_location.proto**: A comprehensive protocol that merges various geographical parameters.
+- **traveller.proto**: Profiles, preferences, and details of travellers.
+- **travel_period.proto**: Specifies the time frame for travel plans, vacations, or any related temporal span.
+
+Whether you are integrating the Camino Message Types into your platform, extending its capabilities, or simply learning more about our approach to digital travel solutions, this documentation aims to be your comprehensive guide. We encourage developers and enthusiasts alike to explore and contribute, ensuring that the Camino Message Types remains at the forefront of innovation.
+
+## Camino Messenger Protocol API Documentation
+
+Camino Messenger Protocol is published on [buf.build](https://buf.build/chain4travel/camino-messenger-protocol/docs/dev). You can find below direct links to different packages in the Camino Messenger Protocol definition.
+
```

Comment:
On buf.build we have 3 branches: 

- main: is the latest official Message Type release and the below links refer to the main branch.
- dev: that might contain work under development, which can be used by partners who are aware of the new feature and are participating in its development.
- draft: used to sketch ideas andobtain feedback from partners before development starts